### PR TITLE
SDX-1144 Changed Delete to Archive

### PR DESCRIPTION
**Pull request by mohsinhijazee**

### CODE REVIEW

File: src/framework/statusDropdown/StatusDropdown.jsx [hunk 1]
```diff
@@ -42,13 +42,13 @@ export default class StatusDropdown extends Component {
     this.optionToNameMap = {
       [StatusDropdown.OPTIONS.ACTIVATE]: 'Start',
       [StatusDropdown.OPTIONS.DEACTIVATE]: 'Pause',
-      [StatusDropdown.OPTIONS.DELETE]: 'Delete',
+      [StatusDropdown.OPTIONS.ARCHIVE]: 'ARCHIVE',
```
Comment (ahollesen) [inline src/framework/statusDropdown/StatusDropdown.jsx]: Order and not in upper case.

File: src/framework/statusDropdown/StatusDropdown.jsx [hunk 2]
```diff
@@ -42,13 +42,13 @@ export default class StatusDropdown extends Component {
...
     };
 
     this.optionToSelectedNameMap = {
       [StatusDropdown.OPTIONS.ACTIVATE]: 'Running',
       [StatusDropdown.OPTIONS.DEACTIVATE]: 'Paused',
-      [StatusDropdown.OPTIONS.DELETE]: 'Deleted',
+      [StatusDropdown.OPTIONS.ARCHIVE]: 'ARCHIVEd',
```
Comment (ahollesen) [inline src/framework/statusDropdown/StatusDropdown.jsx]: Order and not in upper case.

File: src/framework/statusDropdown/StatusDropdown.jsx [hunk 3]
```diff
@@ -148,7 +148,7 @@ export default class StatusDropdown extends Component {
 StatusDropdown.OPTIONS = keyMirror({
   ACTIVATE: null,
   DEACTIVATE: null,
-  DELETE: null,
+  ARCHIVE: null,
```
Comment (ahollesen) [inline src/framework/statusDropdown/StatusDropdown.jsx]: Order.

File: src/framework/statusDropdown/StatusDropdown.jsx [hunk 4]
```diff
@@ -31,7 +31,7 @@ export default class StatusDropdown extends Component {
       [StatusDropdown.OPTIONS.ACTIVATE]: StatusDropdownOptionIcon.TYPE.ACTIVATE,
```
Comment (ahollesen) [inline src/framework/statusDropdown/StatusDropdown.jsx]: I'd fill out the other maps alright grey is the default color but it makes it way clearer.

File: src/framework/statusDropdown/statusDropdownOptionIcon/StatusDropdownOptionIcon.jsx [hunk 5]
```diff
@@ -9,7 +9,7 @@ const StatusDropdownOptionIcon = (props) => {
   const classMap = {
     [StatusDropdownOptionIcon.TYPE.ACTIVATE]: 'icon-play-blue',
     [StatusDropdownOptionIcon.TYPE.DEACTIVATE]: 'icon-pause-blue',
-    [StatusDropdownOptionIcon.TYPE.DELETE]: 'icon-remove-circle-blue',
+    [StatusDropdownOptionIcon.TYPE.ARCHIVE]: 'icon-remove-circle-blue',
```
Comment (ahollesen) [inline src/framework/statusDropdown/statusDropdownOptionIcon/StatusDropdownOptionIcon.jsx]: Order.

File: src/framework/statusDropdown/statusDropdownOptionIcon/StatusDropdownOptionIcon.jsx [hunk 6]
```diff
@@ -26,7 +26,7 @@ const StatusDropdownOptionIcon = (props) => {
 StatusDropdownOptionIcon.TYPE = keyMirror({
   ACTIVATE: null,
   DEACTIVATE: null,
-  DELETE: null,
+  ARCHIVE: null,
```
Comment (ahollesen) [inline src/framework/statusDropdown/statusDropdownOptionIcon/StatusDropdownOptionIcon.jsx]: Order.

**Comment by mohsinhijazee** [comment]
Sorry, this wasn't supposed to go for code review along - was in context of the main feature ticket. My bad.

**Pull request closed without merging**